lib/perl/lib/Apache/TS/AdminClient.pm

```diff
@@ -45,10 +45,6 @@ use constant {
     TS_EVENT_REG_CALLBACK   => 12,
     TS_EVENT_UNREG_CALLBACK => 13,
     TS_EVENT_NOTIFY         => 14,
-    TS_SNAPSHOT_TAKE        => 15,
-    TS_SNAPSHOT_RESTORE     => 16,
-    TS_SNAPSHOT_REMOVE      => 17,
-    TS_SNAPSHOT_GET_MLT     => 18,
     TS_DIAGS                => 19,
     TS_STATS_RESET          => 20
```

Comment:
Should these get renumbered?

Reply:
Renumbering would theoretically break compatibility though. But, that could be ok here, since it's only in a major release.

Reply:
When you removed them from the enum, they get re-ordered though. This is just the perl admin client that should match and no longer will.